modell/src/main/kotlin/no/nav/tilbakekreving/behandling/saksbehandling/Foreldelsesteg.kt

```diff
@@ -0,0 +1,30 @@
+package no.nav.tilbakekreving.behandling.saksbehandling
+
+import java.math.BigDecimal
+import java.time.LocalDate
+import no.nav.tilbakekreving.api.v1.dto.VurdertForeldelseDto
+import no.nav.tilbakekreving.api.v1.dto.VurdertForeldelsesperiodeDto
+import no.nav.tilbakekreving.kontrakter.behandlingskontroll.Behandlingssteg
+import no.nav.tilbakekreving.kontrakter.foreldelse.Foreldelsesvurderingstype
+import no.nav.tilbakekreving.kontrakter.periode.Datoperiode
+
+class Foreldelsesteg : Saksbehandlingsteg<VurdertForeldelseDto> {
+    override val type: Behandlingssteg = Behandlingssteg.FORELDELSE
+
+    override fun erFullstending(): Boolean = true
+
+    override fun tilFrontendDto(): VurdertForeldelseDto {
+        return VurdertForeldelseDto(
+            foreldetPerioder = listOf(
+                VurdertForeldelsesperiodeDto(
+                    periode = Datoperiode(LocalDate.now().minusYears(3), LocalDate.now().minusYears(3).plusMonths(2)),
```

Comment:
Dette blir ikke litt bedre da?
```suggestion
                    periode = Datoperiode(LocalDate.now().minusYears(3), LocalDate.now().minusYears(2).minusMonths(10)),
```